api-report/container-utils.api.md

```diff
@@ -6,11 +6,12 @@
 
 import { ICriticalContainerError } from '@fluidframework/container-definitions';
 import { IErrorBase } from '@fluidframework/container-definitions';
+import { IFluidErrorBase } from '@fluidframework/telemetry-utils';
```

Comment:
I imagine I should have said something elsewhere/earlier but I'll just share my thought passing by "IFluidErrorBase" here anyway... why "Base" - I think it is an awkward term to postpend to the identifier of an interface. What is being conveyed? That we would expect the type that satisfies this interface to be a part of an inheritance hierarchy?

Reply:
Sure, I was going off existing `IErrorBase` and `IDriverErrorBase`.  Not going to change it today.